language/diem-tools/writeset-transaction-generator/src/main.rs

```diff
@@ -68,25 +126,78 @@ fn main() -> Result<()> {
             &serde_json::from_str::<serde_json::Value>(args.as_str())?,
             execute_as,
         ),
-        Command::CreateDiemFrameworkRelease { url, version } => {
-            let remote = JsonRpcDebuggerInterface::new(url.as_str())?;
-            let remote_modules = remote.get_diem_framework_modules_by_version(version)?;
-            let local_modules = build_stdlib().into_iter().map(|(_, m)| m).collect();
-            let payload = create_release_writeset(remote_modules, local_modules)?;
-            verify_payload_change(&remote, Some(version), &payload)?;
+        Command::CreateDiemFrameworkRelease {
+            chain_id,
+            url,
+            version,
+        } => {
+            if let Ok(artifact) = load_artifact(&chain_id) {
```

Comment:
It might be better to use `load_artifact?` here and manually create release `version` 0 artifacts for each chain ID?

Reply:
Added a maunal flag to create the artifact for the first release.